purchase_quotation_products/views/product_product_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_product_tree_view" model="ir.ui.view">
+        <field name="name">product.product.form</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_product_tree_view"/>
+        <field name="mode">primary</field>
+        <field name="priority">40</field>
+        <field name="arch" type="xml">
+            <tree position="attributes">
+                <attribute name="edit">true</attribute>
+                <attribute name="create">false</attribute>
+                <attribute name="editable">top</attribute>
+            </tree>
+            <xpath expr="//tree">
+                <button name="action_product_form" type="object" icon="fa-external-link" string="Open Product Form View"/>
+            </xpath>
+             <xpath expr="//tree//field[1]" position="before">
+                <field name="qty_purchase"/>
+            </xpath>
+            <field name="lst_price" position="replace">
+                <field name="standard_price" readonly="1"/>
+            </field>
+            <field name="default_code" position="attributes">
+                <attribute name="readonly">1</attribute>
+            </field>
+            <xpath expr="//field[@name='name']" position="attributes">
```

Comment:
@nicomacr 
Please apply the same change to the other elements of the view as you have been applied to the `//field[@name='name']`
 
![image](https://user-images.githubusercontent.com/7593953/41852789-7e7e0f3e-7862-11e8-9124-2639fe42214f.png)